test/java/seedu/addressbook/common/UtilsTest.java

```diff
@@ -36,11 +36,34 @@ public void elementsAreUnique() throws Exception {
         assertNotUnique(null, "a", "b", null);
     }
 
+    @Test
+    public void elementsContainsNull() {
+        Object nullObj = null;
+        // some object contains null
+        assertContainsNull("", 1, new ArrayList<>(), nullObj, "cc");
+        assertContainsNull("", 1, new ArrayList<>(), nullObj);
+        assertContainsNull(nullObj);
+        assertContainsNull("a", nullObj);
+
+        // all objects must not contain null
+        assertNoNull("");
+        assertNoNull(1);
+        assertNoNull(1, "abc", new ArrayList<>(), 2, 1.0f, "bbb");
+    }
+
     private void assertAreUnique(Object... objects) {
         assertTrue(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
 
     private void assertNotUnique(Object... objects) {
         assertFalse(Utils.elementsAreUnique(Arrays.asList(objects)));
     }
+
+    private void assertContainsNull(Object ... objects) {
```

Comment:
Extracting the assert function is not really necessary here since this method is quite simple (with only one line). Thus, you can add it directly into your test cases.